src/assets/templates/academics_graduation_checklist.html

```diff
@@ -17,7 +17,7 @@ <h2 class="cc-left">Graduation Checklist</h2>
               Your expected graduation term is: <strong><span data-ng-bind="graduation.undergraduate.expectedGraduationTerm.termName"></span></strong>
             </div>
             <div class="cc-widget-padding">
-              It is expected that you complete the following before graduation.<br><br>
+              It is expected that you complete the following before graduation;<br><br>
```

Comment:
@redconfetti I think this should be a colon `:` not a `;`

>replace . with : at the end of sentence "It is expected..."

Reply:
LOL, I guess even the simplest of tasks can require correction in code review. 🤦